dev-support/ci/selective_ci_checks.sh

```diff
@@ -242,18 +242,25 @@ function get_count_doc_files() {
     start_end::group_end
 }
 
-function get_count_junit_files() {
-    start_end::group_start "Count junit test files"
+function get_count_integration_files() {
+    start_end::group_start "Count integration test files"
     local pattern_array=(
         "^hadoop-ozone/dev-support/checks/_mvn_unit_report.sh"
         "^hadoop-ozone/dev-support/checks/integration.sh"
-        "^hadoop-ozone/dev-support/checks/unit.sh"
+        "^hadoop-ozone/integration-test"
+        "^hadoop-ozone/fault-injection-test/mini-chaos-tests"
         "src/test/java"
-        "src/test/resources"
+    )
+    local ignore_array=(
+        "^hadoop-hdds/.*/src/test/java/.*/Test.*.java"
+        "^hadoop-ozone/[a-eghj-z].*/src/test/java/.*/Test.*.java"
```

Comment:
This regex is a bit puzzling, at first.

Perhaps a comment like:
#capture all directories except the "integration-test" directory
?

Reply:
Thanks @GeorgeJahad for the reminder on adding comment.

How about this?

```
    # Ozone's unit test naming convention: Test*.java
    # The following makes this filter ignore all tests except those in
    # integration-test and fault-injection-test.
    # Directories starting with `i` under hadoop-ozone need to be listed
    # explicitly, other subdirectories are captured by the second item.
    local ignore_array=(
        "^hadoop-hdds/.*/src/test/java/.*/Test.*.java"
        "^hadoop-ozone/[a-eghj-z].*/src/test/java/.*/Test.*.java"
        "^hadoop-ozone/insight/src/test/java/.*/Test.*.java"
        "^hadoop-ozone/interface-client/src/test/java/.*/Test.*.java"
        "^hadoop-ozone/interface-storage/src/test/java/.*/Test.*.java"
    )
```

Reply:
looks good to me.